pkg/parser/parser.peg

```diff
@@ -369,18 +369,22 @@ StandaloneAttributes <- attributes:(ElementAttribute)+ BlankLine* EOF { // stand
 
 QuotedString <- SingleQuotedString / DoubleQuotedString
 
-SingleQuotedString <- "'`" elements:SingleQuotedStringElements "`'" {
+SingleQuotedString <- SingleQuoteStringStart elements:SingleQuotedStringElements SingleQuoteStringEnd {
     return types.NewQuotedString(types.SingleQuote, elements.([]interface{}))
 }
 
 SingleQuotedStringElements <- elements:(SingleQuotedStringElement+) {
     return types.NewInlineElements(elements)
 }
 
+SingleQuoteStringStart <- "'`" ![ \t\r\n]
```

Comment:
you should be able to replace `![ \r\n\t]` with `[^ \r\n\t]`

Reply:
No I think that is wrong.  I don’t want to consume the token.  This leaves the next tokens available in the input stream without consuming them.

Reply:
This is important for nesting quotes text for example.

Reply:
ah right! I think that `&[^ \r\n\t]` would work in that case then. I like the `&` syntax because it really says that it looks forward in the content without consuming the next character.

Reply:
I think the two syntaxes are basically the same.  Using ! should convey that as well as & with one fewer characters.  I guess it’s all what you’re used to.

Reply:
ok, I can try and refactor that later if you don't want to make the change here. It's not a big deal indeed.

Reply:
Actually there might be a difference - what happens with EOF.  Though that difference won’t matter in this particular case it might in others. 

More precisely the ! syntax says that the element is NOT followed by one of those characters whereas the & says that it IS followed by a character that is not in that list.

Reply:
yes, I believe it's fine here, but it matters for delimited blocks for which the ending delimiter can be omitted when the block is at the end of the document. (Asciidoc is permissive on that regards.)